src/org/loklak/http/ClientConnection.java

```diff
@@ -272,7 +273,7 @@ public static String getRedirect(String urlstring, boolean useAuthentication) th
                 throw new IOException("redirect for  " + urlstring+ ": no location attribute found");
             } else {
                 EntityUtils.consumeQuietly(httpEntity);
-                throw new IOException("no redirect for  " + urlstring+ " fail: " + httpResponse.getStatusLine().getStatusCode() + ": " + httpResponse.getStatusLine().getReasonPhrase());
+                return urlstring;
```

Comment:
This is to understand redirects, Can you explain why it doesn't throw an `IOException` anymore and instead can you ensure it always returns the `urlstring`?